doc/security/specs/attestation/_index.md

```diff
@@ -0,0 +1,879 @@
+    *   `pathLenConstraint` not included.
+
+OpenTitan Owner Identity custom extension:
+
+*   `extnID`: TBD. Need to determine if we need an OID for OpenTitan, or if we
+    can allocate a new ID under the Google certificate extension domain
+    1.3.6.1.4.1.11129.2.1.
+*   `critical`: FALSE
+*   `extnValue`: See [owner custom extension](#owner-custom-extension) section.
+
+This certificate may also contain the
+[creator custom extension](#creator-custom-extension) if the Owner decides to
+endorse this certificate with its own PKI.
+    </td>
+  </tr>
+</table>
+
+#### Owner Custom Extension
+
+ASN.1 SEQUENCE containing the following fields. The owner can extend this as
+needed.
+
+<table>
+  <tr>
+    <td><strong>Seq. No.</strong></td>
+    <td><strong>Name</strong></td>
+    <td><strong>Encoding</strong></td>
+    <td><strong>Description</strong></td>
+  </tr>
+  <tr>
+    <td>0</td>
+    <td>Code Descriptor</td>
+    <td>OCTET STR</td>
+    <td>BL0 version and key manager software binding tag information.</td>
+  </tr>
+</table>
+
+## Lightweight Certificate Format
+
+The Silicon Creator may opt to implement attestation signatures over a binary
+manifest containing the endorsed key as well as the measurements described in
+the [creator](#creator-custom-extension) and [owner](#owner-custom-extension)
+custom extensions.
+
+The binary manifest format must be standardized at the OpenTitan project level
+to ensure interoperability between silicon and software layers.
+
+## Privacy Considerations {#privacy-considerations}
+
+The Silicon Owner software must address any privacy concerns associated with the
+use of device identifiable information.
+
+The Silicon Owner certificate, for example, is meant to be used exclusively with
+the Silicon Owner’s infrastructure. Use cases requiring multiple identities or
+attestation services for multiple backends are required to derive
+<span style="text-decoration:underline;">application level identities and
+certificates</span>. Such application level identities must fulfill the privacy
+requirements of their use cases.
+
+OpenTitan’s key manager provides a mechanism to derive multiple application
+level keys with lineage to the Owner Identity, without making them
+distinguishable from any random key. These properties allow the software
+implementation to bind the OpenTitan attestation to any application level
+attestation flows without precluding the implementation of additional privacy
+measures at the application level.
+
+
+<!-- Footnotes themselves at the bottom. -->
+## Notes
+
+[^1]: In-field ROM_EXT updates performed by a Silicon Owner may invalidate the Creator Identity and its associated certificate. Without a federated device registry, new silicon owners are left to trust the Creator Identity certificate provided by the current owner to verify the authenticity of the device.
+
+[^2]: The Creator Identity is referred to as the Unique Device Secret (UDS) in DICE terminology.
```

Comment:
nit: These can be wrapped, I tested locally.

Reply:
Done.